confgenerator/add_lua_script.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package confgenerator
+
+import (
+  "fmt"
+  "io/ioutil"
+  "os"
+  "path/filepath"
+)
+
+const addLogNameLuaScriptContents string = `
+function split (inputstr, sep)
```

Comment:
[No action needed]

Just surprised that LUA does not provide a built-in string splitting function LOL

Reply:
Yeah I was surprised too actually :/